docs/evaluation/convolution.rst

```diff
@@ -92,26 +93,26 @@ since it has sharp edges and so the convolution is more obvious).
     >>> point.xlow = 9.5
     >>> point.xhi = 10.5
     >>> print(point)
-    point
+    pt
```

Comment:
It would probably have been easier to change the earlier line to

    >>> point = Box1D('point')

but you'd still have to change the `convolved = psf1(point)` line below.

I'm not saying that you should do this (I think re-iterating the fact that the model name and the variable name do not need to be the same thing has some value!), just musing aloud.

Reply:
I'll think about it...